data/assets/nightsky/zeropointdata.csv

```diff
@@ -0,0 +1,2 @@
+x,y,z,none
```

Comment:
This file is now available under the identifier `zeropoint_data` so we can remove it from the repository